pydax/_high_level.py

```diff
@@ -234,8 +232,7 @@ def _get_schemata() -> SchemaManager:
 
     global _schemata
 
-    if _schemata is None:
```

Comment:
Why do we want to remove this line?

Reply:
In the case where `init` is called to update a schema URL using `update_only=True`, `_global_config` gets updated, but `_schemata` keeps the old schema. 

Removing that line ensures the `else` block in `load_schemata` gets called:
```python
else:
    for name, schema in _schemata.schemata.items():
        if schema.retrieved_url_or_path != urls[name]:
            _schemata.add_schema(name, Schema(urls[name]))
```
which updates `_schemata`. Originally I considered setting `_schemata = None` in the `if` block of `init` if a schema URL was updated, but that would require reloading all schemata even if just one was updated. Not sure if you have a better way in mind here?

Reply:
That's a good point and is some bug that we didn't discover. Mind adding a test for this?

Reply:
sure thing, added